### PR TITLE
Release doc

### PR DESCRIPTION
**Pull request by maleck13**
@david-martin @jasonmadigan review pls

### CODE REVIEW

File: process.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+
+The following components are needed to make up a integreatly release
+
+- Component: Installer
+    - **Location:** https://github.com/integr8ly/installation
+    - **Role:** Setup and installation to run against an OpenShift cluster to install the required products and components
+    - **Releasing:** 
+        - create a release branch named `vx.x.x`
+        - any bug fixes in the installer should be done against master and cherry picked to the release branch
```
Comment (david-martin) [inline process.md]: are release branches forever living, or should the last commit be tagged & branch deleted after the release is finalised?

File: process.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- Component: Managed Service Broker
+    - **Location:** https://github.com/integr8ly/managed-service-broker
+    - **Role:** Open Service Broker API implementation that allows us to provision services in a different namespace than the one where the provisioned service
+    was created. As much as possible, it will use CRs and operators to set up these services.
+    - **Releasing:** 
+        - tag the managed service repo against master with the version we want to release.
+        - clone the repo and checkout the tag
+        - run ```make build_and_push TAG=<release_version>```
+        - update the version referenced from the integreatly installer against the release branch [https://github.com/integr8ly/installation]()
+        - for each bug fix we will need to either update the tag or update the version retag and update the installer
```
Comment (david-martin) [inline process.md]: Steps here sound reasonable, but maybe a branch would work better to allow bug fixes.
Should we move these steps to the managed service broker repo and link to them here?

File: process.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        
+        
+- Component: Integreatly Web App
+    - **Location:** https://github.com/integr8ly/tutorial-web-app
+    - **Role:** Web Application that walks customers through the different types of integrations they can perform on the integreatly cluster
+    - **Releasing:** 
+        - follow the instructions here [https://github.com/integr8ly/tutorial-web-app#releasing]
+        - update the version referenced in the integreatly installer release branch
+        - for each bug fix we will need to either update the tag or update the version retag and update the installer
```
Comment (david-martin) [inline process.md]: Releasing a new version of the webapp if there are bugfixes sounds like the right move here.
It's an independent component.

File: process.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        - update the version referenced from the integreatly installer against the release branch [https://github.com/integr8ly/installation]()
+        - for each bug fix we will need to either update the tag or update the version retag and update the installer
+        
+        
+- Component: Integreatly Web App
+    - **Location:** https://github.com/integr8ly/tutorial-web-app
+    - **Role:** Web Application that walks customers through the different types of integrations they can perform on the integreatly cluster
+    - **Releasing:** 
+        - follow the instructions here [https://github.com/integr8ly/tutorial-web-app#releasing]
+        - update the version referenced in the integreatly installer release branch
```
Comment (david-martin) [inline process.md]: `update the version referenced in the integreatly installer release branch`
What does this mean?
Update a var in a yml file in the installation repo?

File: process.md (hunk 1 above)
Comment (maleck13) [reply process.md]: Tagging the last commit and deleting the branch sounds reasonable. I will update

File: process.md (hunk 4 above)
Comment (maleck13) [reply process.md]: yes

**Pull request merged**